sui/utils/sign-utils.js

```diff
@@ -75,10 +81,18 @@ function getSuiClient(chain, rpc) {
 }
 
 async function printWalletInfo(wallet, client, chain, options = {}) {
-    const owner =
-        wallet instanceof Ed25519Keypair || wallet instanceof Secp256k1Keypair || wallet instanceof Secp256r1Keypair
-            ? wallet.toSuiAddress()
-            : wallet;
+    let owner;
+
+    if (options.privateKey !== 'ledger') {
+        owner =
+            wallet instanceof Ed25519Keypair || wallet instanceof Secp256k1Keypair || wallet instanceof Secp256r1Keypair
+                ? wallet.toSuiAddress()
+                : wallet;
```

Comment:
When is it not an instance of a KeyPair? Supporting the latter case seems weird, even though this was being done before